baybe/utils/dataframe.py

```diff
@@ -361,6 +361,50 @@ def df_uncorrelated_features(
     return data
 
 
+def df_add_noise_to_degenerate_rows(
+    df: pd.DataFrame, noise_level: float = 0.001
+) -> pd.DataFrame:
+    """Add noise to degenerate rows to being able to numerically distinguish them.
```

Comment:
perhaps a bit too much "to" 😄 
```suggestion
    """Add noise to degenerate rows to make them numerically distinguishable.
```

Reply:
added